### PR TITLE
Restyle sidebar rendered buttons

### LINKED ISSUE #56: Restyle sidebar rendered buttons
**Is your feature request related to a problem? Please describe.**
This issue will make sure to keep all buttons rendered on the sidebar are consistent 

**Describe the solution you'd like**
Styled components and paul magic

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Restyle the rendered buttons
2. Added headings to each added group 

## Purpose
UI clean up

## Approach
Styled components

Closes #56

### CODE REVIEW

File: src/components/IngredientButton.tsx [hunk 1]
```diff
@@ -35,15 +57,19 @@ const IngredientButton: FC<IngredientButtonProps> = ({
...
-        <RemoveBtn
-          src="close.svg"
-          onClick={() => removeIngredient(ingredientName)}
-        ></RemoveBtn>
+        <IngredientNameContainer>
+          <h3>
+            {ingredientName}
+          </h3>
+        </IngredientNameContainer>
+        {/* <RemoveBtn src="close.svg" onClick={()=>removeIngredient(ingredientName)}></RemoveBtn> */}
```
Comment (BlackCubes) [inline src/components/IngredientButton.tsx:67]: Do ye need dis still??

**Review by BlackCubes — APPROVED** [review]
Looks gewd! Just remove the comment, if can be

File: src/components/IngredientButton.tsx (hunk 1 above)
Comment (Lap343) [reply src/components/IngredientButton.tsx:67]: naughty naughty

Comment (gonzalespaulb) [reply src/components/IngredientButton.tsx:67]: Good catch!

**Review by Lap343 — APPROVED** [review]
When you remove the commented code this PR is good to go

**Pull request merged**